plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -674,8 +675,9 @@ public void testCreateTableAs()
     @Test
     public void testIdentifiers()
     {
-        session.execute("DROP KEYSPACE IF EXISTS \"_keyspace\"");
-        session.execute("CREATE KEYSPACE \"_keyspace\" WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor': 1}");
+        String quotedKeyspaceNameStartingWithUnderscore = "\"_keyspace\"";
```

Comment:
Could you add random suffix? This test will be flaky if it runs concurrently in SaaS environment.